elastic/kibana
# [Console Monaco] Fix autocomplete insert text

Fixes https://github.com/elastic/kibana/issues/188192

## Summary

This PR fixes the scenario where an autocomplete suggestion is inserted with an extra quote in the beginning.

**How to test:**
1. Go to Console
2. Insert the following request
```
GET /_search
{
    "query": {
      "match_all": {}
    }
}
```
3. Put the cursor at the end of the `match_all` field (right before the closing quote) and then delete a few of the last characters.
4. Retype one character in order to get the suggestions popup displayed. Then press Enter to add a suggestion.
5. Verify that the suggestion is added with no extra quote in the beginning.
6. You can also test with different fields and indentations. For example:
```
GET /_search
{"query": {"match_all": {}}}
```

https://github.com/user-attachments/assets/7f4336a9-2330-445f-ab06-2d02ba2ef37d
